test/hydro/volume_of_the_submerged_body_part_from_body_density_and_liquid_density_test.py

```diff
@@ -13,8 +13,8 @@
 
 @fixture(name="test_args")
 def test_args_fixture():
-    body_volume = Quantity(0.04 * units.meter**3)
-    body_density = Quantity(2600 * units.kilogram / units.meter**3)
+    body_volume = Quantity(12 * units.meter**3)
+    body_density = Quantity(500 * units.kilogram / units.meter**3)

```

Comment:
You should not simply dismiss a valid test case. Previous numbers were valid but gave bizarre result. Could you explain it?